scripts/fulltext.py

```diff
@@ -0,0 +1,204 @@
+import argparse
+import json
+import os
+import psycopg2
+from glob import glob
+from datetime import datetime
+from sqlalchemy.orm import sessionmaker, load_only
+from sqlalchemy import create_engine, text
```

Comment:
The script is not using this imports, and it is possible that they are not present in the container either so the script might fail with an exception.